projects/packages/forms/src/blocks/field-telephone/index.js

```diff
@@ -22,9 +22,32 @@ const settings = {
 		src: <Icon icon={ mobile } />,
 	},
 	edit,
+	attributes: {
+		...defaultSettings.attributes,
+		showCountrySelector: {
+			type: 'boolean',
+		},
+		default: {
+			type: 'string',
+		},
+		onChangeDefaultCountry: {
```

Comment:
Why is it necessary to pass a function in attributes? Is it interactivity API thing?